src/services/document-service.js

```diff
@@ -416,9 +418,24 @@ class DocumentService {
       contributors,
       tags: lastRevision.tags,
       archived: lastRevision.archived,
-      origin: lastRevision.origin
+      origin: lastRevision.origin,
+      cdnResources: lastRevision.cdnResources
     };
   }
+
+  _getCdnResources(sections) {
+    return [
+      ...sections.reduce((cdnResources, section) => {
+        const info = getPluginInfoByType(section.type);
```

Comment:
But shouldn't we fix this on each plug-in info file? What guarantees that that function will always be used only here and that we will remember about the faulty null content trouble? I'd consider that the getCdnResources is still buggy or?

Reply:
As soon as it's deleted, a section is not handed down to the plugin code anymore (e.g. `display`, `renderer`, `cloneContent`), so the idea was to stick to that. But I'm also not super sure about it.

Reply:
My question is how are you sure that this will only be used here in the future? If we use this thing somewhere else we might forget that the content might be null...

Reply:
We can't be sure, but calling a plugin info function on a deleted section I would consider an error in the business logic (see also the `cloneContent` function in the same file, that won't work either with deleted sections). Nevertheless, I'll change as it's not gonna hurt. I'd like to leave the check for "deleted-ness" in the service though.

Reply:
Ok, I don't have a problem with the check. Maybe then it's better to leave it as it is now so we get a problem rather than hiding it.

Reply:
That's also a good point, may rather have the error when you're not supposed to call it. As I said, I wasn't sure about this either. If you're ok with it, I'd leave it like it is for now.